packages/react-sdk/README.md

```diff
@@ -0,0 +1,91 @@
+# Bucket React SDK
+
+React client side library for [Bucket.co](https://bucket.co)
+
+## Install
+
+Install via npm:
+
+```
+npm i @bucketco/react-sdk
+```
+
+## Setup
+
+Wrap your application with the `Bucket` higher order component.
+
+This will initialize Bucket, fetch feature flags and start listening for Live Satisfaction events.
+
+```tsx
+<Bucket publishableKey="{YOUR_PUBLISHABLE_KEY}">{/* ... */}</Bucket>
+```
+
+### Props
+
+All options which can be passed to `bucket.init` can be passed as props to the Bucket higher order component.
```

Comment:
probably worth adding a link to the readme there. Don't want to repeat everything here, for example LiveSatisfaction configuration.